src/app/systelab-components/loading/README.md

```diff
@@ -2,11 +2,32 @@
 
 Component to show a loading wheel.
 
+Sometimes when user submits a form or the application is loading is needed to disable the content of the whole application in order to avoid user interactions with the UI while the process of the submit or loading ends.
+
 ## Using the template
 
-```
-<systelab-loading></systelab-loading>
-```
+Systelab loading component is usually set in the main template of the application in order to disable the whole application. To viwe loading 
+
+Exists a service called LoadingService to be injected in the components. LoadingService has the folllowing methods:
```

Comment:
LoadingService is a service to be injected in order to invoke the following methods.